client/state/selectors/test/get-site-url.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteUrl } from '..';
+
+describe( 'getSiteUrl()', () => {
+	it( 'should return null if the site is unknown', () => {
```

Comment:
Should we also test the case with a missing `URL` on an existing site object? I'm asking because the selector description contains that case, which would imply testing it.